cli/src/pcluster/validators/ebs_validators.py

```diff
@@ -119,6 +121,13 @@ def _validate(self, volume_type, volume_size, volume_iops):
                     ),
                     FailureLevel.ERROR,
                 )
+        if volume_type in EBS_VOLUME_TYPE_NOT_SUPPORT_IOPS:
+            self._add_failure(
+                "The parameter IOPS is not supported for {0} volumes.".format(
```

Comment:
I'd use an `f"The parameter IOPS is not supported for {volume_type} volumes."` parameter here to save a few lines.

Reply:
Changed